shared/folders/install/banner.js

```diff
@@ -0,0 +1,29 @@
+// @flow
+import React, {Component} from 'react'
+import {connect} from 'react-redux'
+import Install from './banner-install'
+import Uninstall from './banner-uninstall'
+
+import type {TypedState} from '../../constants/reducer'
+
+type Props = {
+  installed: boolean,
+}
+
+class Banner extends Component<void, Props, void> {
```

Comment:
🐑 
this can be replaced with:

`compose(connect(mapStateToProps), branch(props => installed, renderComponent(Uninstall)))(Install)`

grep renderComponent for examples